src/main/groovy/org/grails/plugins/web/Java8GrailsPlugin.groovy

```diff
@@ -70,6 +83,28 @@ This plugin provides support for Java 8 specific functions in a Grails applicati
 
     }}
 
+    /**
+     * Get the configured date formats.
+     *
+     * Tries to:
+     *  1. Load any configured date formats
+     *  2. Falls back to the {@link #DEFAULT_JSR310_FORMATS} for sensible defaults
+     *
+     * @return a list of formats to use when binding dates via {@link Jsr310ConvertersConfiguration}
+     */
+    List<String> getDateFormats() {
+        Field formats
+        try {
+            formats = DataBindingGrailsPlugin.getDeclaredField('DATE_FORMATS')
+        } catch (NoSuchFieldException e) {
+            formats = Settings.getDeclaredField('DATE_FORMATS')
+        }
+        String dateFormatProperty = formats.get(null)
+
+        List dateFormats = config.getProperty(dateFormatProperty, List, DEFAULT_JSR310_FORMATS)
+        return dateFormats
```

Comment:
Lines 104 and 105 can be simplified to just be the `config.getProperty` call

Reply:
Will collapse and include pending other discussion re `Set`